### ISSUE TITLE
[BUG]  Validator Index wrong in ethereum_staking.flows

### ISSUE THREAD

**Issue by Yonatank88** [duneanalytics/spellbook#6101]
### Description
Some where around validator index 1023000 there seems to start a discrepancy between validator index and public key
### Current behavior
Run the following query:
```
SELECT
  *
FROM staking_ethereum.flows
WHERE
  validator_index = 1023000
```

Copy the public key and search for it on [beaconcha.in ](https://beaconcha.in/validator/957d8b0b7d3dde6c8c9dfd7980f5a8a4043fe6c6d5ac10ea78c7f578764f5fadb080a9c1b52252b0fa30f831a1527252)
Youll see  the validator the public key has a different index then in the query response

### Expected behavior
Validator index should be identical in dune and beaconcha.in

### Impacted model

[ethereum_staking.flows](https://github.com/duneanalytics/spellbook/blob/3ef2b140c0bff98980d7ca0174e1ded1ad14595e/models/staking/ethereum/staking_ethereum_flows.sql#L11) Invalid public keys list 

[ethereum_staking.flows](https://github.com/duneanalytics/spellbook/blob/3ef2b140c0bff98980d7ca0174e1ded1ad14595e/models/staking/ethereum/staking_ethereum_flows.sql#L25) index calculation

### Possible Solution

My best guess is that there are a few more invalid public keys that should be added so they won't be counted in the index counter but I'm not sure whats the right way to find invalid validators.

**Comment by jeff-dude** [comment]
@hildobby did you want to look into this?

**Comment by hildobby** [comment]
> @hildobby did you want to look into this?

yes its on my backlog @jeff-dude, thanks for pointing out @yoanatanblox 🙏🙏
